dev-docs/howtoguides/how_to_use_magic_attach_endpoints.md

```diff
@@ -0,0 +1,165 @@
+# How to use magic attach API endpoints
+
+Right now, the pro client provides three distinct endpoints to make it easier to perform
```

Comment:
The docs are intended to always be up to date, so I don't think we need the "Right now"
Also `pro` isn't released yet, and these docs will be immediately available, so we should use `ua`.
```suggestion
The UA Client provides three distinct endpoints to make it easier to perform
```